app/views/home/viewFinalResults.scala.html

```diff
@@ -29,27 +29,29 @@
             <div class="inner-block-padr">
                 <div class="hgroup">
                     <h1 class="heading-xlarge">Home</h1>
-                    <p class="subtitle">@{cachedUsr.user.firstName + " " + cachedUsr.user.lastName}</p>
+                    <p class="subtitle">@{cachedUser.user.firstName + " " + cachedUser.user.lastName}</p>
                 </div>
             </div>
         </div>
-        @if(WithdrawApplicationRole.isAuthorized(cachedUsr)){
         <div class="grid grid-1-2">
             <div class="page-link">
+                @if(!WithdrawnApplicationRole.isAuthorized(cachedUser)) {
```

Comment:
Can you explain this one to me? The way i read it this was:

- If a user is not withdrawn, but can withdraw - show them view and edit

Now i read it as:

- If a user is withdrawn show them view, if the user is not withdrawn but can withdraw show them edit

Users have to not be exported for either check to return true